pkg/providers/apisix/provider_init.go

```diff
@@ -48,23 +48,26 @@ func (p *apisixProvider) Init(ctx context.Context) error {
 		pluginConfigMapA6 = make(map[string]string)
 	)
 
+	p.apisixSharedInformerFactory.Start(ctx.Done())
+	p.apisixSharedInformerFactory.WaitForCacheSync(ctx.Done())
+
 	namespaces := p.namespaceProvider.WatchingNamespaces()
+
 	for _, key := range namespaces {
 		log.Debugf("start to watch namespace: %s", key)
 		wg.Add(1)
 		go func(ns string) {
 			defer wg.Done()
 			// ApisixRoute
-			opts := v1.ListOptions{}
 			switch p.common.Config.Kubernetes.APIVersion {
 			case config.ApisixV2beta3:
-				retRoutes, err := p.common.KubeClient.APISIXClient.ApisixV2beta3().ApisixRoutes(ns).List(ctx, opts)
+				retRoutes, err := p.apisixSharedInformerFactory.Apisix().V2beta3().ApisixRoutes().Lister().ApisixRoutes(ns).List(labels.Nothing())
```

Comment:
```suggestion
				retRoutes, err := p.apisixSharedInformerFactory.Apisix().V2beta3().ApisixRoutes().Lister().ApisixRoutes(ns).List(labels.Everything())
```

Reply:
Thanks for reminding, I'll fix it as soon as possible